plugins/woocommerce/src/Admin/Notes/Note.php

```diff
@@ -229,7 +229,7 @@ public function get_content( $context = 'view' ) {
 	 * Get note content data (i.e. values that would be needed for re-localization)
 	 *
 	 * @param  string $context What the value is for. Valid values are 'view' and 'edit'.
-	 * @return array
+	 * @return object
```

Comment:
$content_data is an object.

https://github.com/woocommerce/woocommerce/blob/63162e65479122d2aedfd44d445402dd921a1f43/plugins/woocommerce/src/Admin/Notes/Note.php#L60